src/main/java/org/jboss/tools/intellij/componentanalysis/CAIntentionAction.java

```diff
@@ -52,7 +53,9 @@ protected CAIntentionAction(PsiElement element, VulnerabilitySource source, Depe
 
     @Override
     public void invoke(@NotNull Project project, Editor editor, PsiFile file) throws IncorrectOperationException {
-        this.updateVersion(project, editor, file, getRecommendedVersion(this.report));
+        String recommendedVersion = getRecommendedVersion(this.report);
+        this.updateVersion(project, editor, file, recommendedVersion);
+        TelemetryService.sendTelemetryEvent(file, recommendedVersion, this.report, "recommendation_accepted");
```

Comment:
```suggestion
        TelemetryService.sendTelemetryEvent(file, recommendedVersion, this.report, "recommendation-accepted");
```
The actionName used dash instead of underscore in other scenarios.
For example: https://github.com/redhat-developer/intellij-dependency-analytics/blob/1b84290776669494558b80a2ccedd82258b6e6d1/src/main/java/org/jboss/tools/intellij/exhort/ApiService.java#L58

Reply:
Initially was using the same action name as Ilona did in the VS Code extension, not sure if they should match.

Reply:
Done.